it/uk/gov/hmrc/excisemovementcontrolsystemapi/GetMessagesControllerItSpec.scala

```diff
@@ -151,45 +152,169 @@ class GetMessagesControllerItSpec extends PlaySpec
       when(movementRepository.getMovementByLRNAndERNIn(any, any))
         .thenReturn(Future.successful(Seq(movementMessage, movementMessage)))
 
-      val result = getRequest
+      val result = getRequest()
 
       result.status mustBe INTERNAL_SERVER_ERROR
     }
 
     "return forbidden (403) when there are no authorized ERN" in {
       withAnEmptyERN()
 
-      getRequest.status mustBe FORBIDDEN
+      getRequest().status mustBe FORBIDDEN
     }
 
     "return a Unauthorized (401) when no authorized trader" in {
       withUnauthorizedTrader(InternalError("A general auth failure"))
 
-      getRequest.status mustBe UNAUTHORIZED
+      getRequest().status mustBe UNAUTHORIZED
     }
 
   }
 
-  private def getRequest = {
+  "GET a message for a movement and a messageId" in {
+    withAuthorizedTrader(consignorId)
+
+    val encodedMessage = Base64.getEncoder.encodeToString(IE801.toString().getBytes(StandardCharsets.UTF_8))
+    val message = Message(encodedMessage, "IE801", messageId, timestamp)
+    when(movementRepository.getMovementById(any))
+      .thenReturn(Future.successful(Some(Movement(validUUID, "boxId", "lrn", consignorId, None, None, Instant.now, Seq(message)))))
+
+    val result = getRequest(messageUrl)
+
+    result.status mustBe 200
```

Comment:
Could use the variable OK instead of 200?

Reply:
done